templates/editor.sidebar.php

```diff
@@ -113,7 +113,7 @@ class="events--button button btn delete btn-half pull-left"
 						</button>
 						<ul class="dropdown-menu" uib-dropdown-menu role="menu" aria-labelledby="single-button">
 							<li role="menuitem"><a ng-click="delete()" href="#"><?php p($l->t('Delete all')); ?></a></li>
-							<li role="menuitem"><a ng-click="deleteOccurrence()" href="#"><?php p($l->t('Delete just occurrence')); ?></a></li>
+							<li role="menuitem"><a ng-click="deleteOccurrence()" href="#"><?php p($l->t('Delete this event')); ?></a></li>
```

Comment:
Maybe we need to add the date and time of the explicit event.
After opening one event you basically get the series editor - the relation to 'this event' is missing

Reply:
You are right! I would suggest that we "fix" the series editor. If you click on any repeating event you will  just see the start and end time of the first event... So every occurrence should have it's own start and end time... But I think that's a way bigger problem because datetimepicker initialization and fullcalendar have no possibility to work together right now 

-> new PR